modules/oceanbase/src/test/java/org/testcontainers/oceanbase/SimpleOceanBaseCETest.java

```diff
@@ -19,9 +19,7 @@ public class SimpleOceanBaseCETest extends AbstractContainerDatabaseTest {
 
     @SuppressWarnings("resource")
     private OceanBaseCEContainer testContainer() {
-        return ((OceanBaseCEContainer) containerProvider.newInstance()).withEnv("MODE", "slim")
-            .withEnv("FASTBOOT", "true")
-            .withLogConsumer(new Slf4jLogConsumer(logger));
+        return ((OceanBaseCEContainer) containerProvider.newInstance()).withLogConsumer(new Slf4jLogConsumer(logger));
```

Comment:
Is this a leftover from debugging?

Reply:
No, as I commented at the Mode enum in the code, the fastboot option was removed as it became the only boot mode for slim deployments. This is a breaking change introduced by the refactoring, and we will avoid such changes in the future.